lib/api/assertions/elementEnabled.js

```diff
@@ -0,0 +1,47 @@
+
+/**
+ * Checks if the given element is visible on the page.
+ *
+ * ```
+ * this.demoTest = function (browser) {
+ *   browser.assert.elementEnabled('.should_be_enabled');
+ *   browser.assert.elementEnabled({selector: '.should_be_enabled'});
+ *   browser.assert.elementEnabled({selector: '.should_be_enabled', suppressNotFoundErrors: true});
+ * };
+ * ```
+ *
+ * @method visible
```

Comment:
I think this should be called just "enabled", that way it will be consistent with "visible". You need to change "visibile" here.

Reply:
Done